Nos/Service/Relay/RelaySubscriptionManager.swift

```diff
@@ -184,16 +183,12 @@ actor RelaySubscriptionManagerActor: RelaySubscriptionManager {
                 } else {
                     activeSubscriptionsCount[relayAddress] = 1
                 }
-            } else if relaySubscription.isOneTime {
-                waitingOneTimeSubscriptions.append(relaySubscription)
-            } else {
-                waitingLongSubscriptions.append(relaySubscription)
             }
+            waitingSubscriptions.append(relaySubscription)
```

Comment:
shouldn't this be inside an `else` block? The way this reads now it looks like we will add active subscriptions to the waiting queue, and then start them again.